packages/hash/frontend/src/blocks/page/PageContext.tsx

```diff
@@ -0,0 +1,33 @@
+import { Schema } from "prosemirror-model";
+import { EditorView } from "prosemirror-view";
+import {
+  createContext,
+  PropsWithChildren,
+  useContext,
+  useMemo,
+  useState,
+} from "react";
+
+interface PageContextProps {
+  editorView: EditorView<Schema> | undefined;
+  setEditorView: (view: EditorView<Schema>) => void;
+}
+
+const PageContext = createContext<PageContextProps>({} as PageContextProps);
```

Comment:
This is a lie – if someone calls `usePageContext` and they're not wrapped in the provider, you can get a runtime crash. 

What you want instead is for the context to store `PageContextProps | null` (and default to `null`) and for `usePageContext` to throw if it encounters `null`.

Reply:
Fixed here https://github.com/hashintel/hash/pull/956/commits/87dd91a552f5e10ceadf71ab4bc62649b09328e8